recipes/libfabric/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "1.22.0" %}
+{% set build = 0 %}
+
+package:
+  name: libfabric
+  version: {{ version }}
+
+source:
+  url: https://github.com/ofiwg/libfabric/releases/download/v{{ version }}/libfabric-{{ version }}.tar.bz2
+  sha256: 485e6cafa66c9e4f6aa688d2c9526e274c47fda3a783cf1dd8f7c69a07e2d5fe
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+
+  run_exports:
+    - {{ pin_subpackage('libfabric', max_pin='x.y') }}
+  ignore_run_exports:
```

Comment:
Moving libuuid to the build section seems to have resolved the issue with the DSO warnings.

Reply:
I think that means libuuid is not being used and you can remove it entirely. From the looks of the configure output and the link check, libuuid is only used by opx and since opx is not enabled, libuuid is not linked at all.

Reply:
OK